<?xml version="1.0"?>
<launch>

	<arg name="limb" default=""/>

    <!-- Phantom node -->
    <include file="$(find omni_common)/launch/omni_new.launch"/>

    <!-- Teleoperation node --> 
    <node pkg="baxter_teleop" type="baxter_phantom_all" name="baxter_teleop_$(arg limb)" output="screen">
      <!-- Axis and buttons parameters -->
      <rosparam file="$(find baxter_teleop)/config/phantom.yaml" command="load" />
      <!-- Twist msg name -->
      <param name="cmd_vel_topic" value="$(arg limb)/cmd_vel" type="string"/>
      <param name="cmd_vel_stmp_topic" value="$(arg limb)/cmd_vel/stamped" type="string"/>
      <param name="cmd_pose_topic" value="$(arg limb)/cmd_pose" type="string"/>
      <param name="cmd_pose_stmp_topic" value="$(arg limb)/cmd_pose/stamped" type="string"/>
      <param name="cmd_joint_topic" value="$(arg limb)/cmd_joint/phantom" type="string"/>
      <param name="haptic_topic" value="phantom/force_feedback" type="string"/>
      <param name="lock_topic" value="phantom/lock" type="string"/>
      <param name="deadman_no_publish" value="false" type="bool"/>
      <remap from="$(arg limb)/ft_sensor_topic" to="/FT_sensor_1/ROS_data"/>
    </node>

</launch>

